spring-cloud-stream-binder-kinesis-docs/src/main/asciidoc/overview.adoc

```diff
@@ -150,6 +150,10 @@ timeToLive::
 See https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/TTL.html[DynamoDB TTL]
 +
 No default - means no records expiration.
+interval::
```

Comment:
You know, looking at this one more time I would say this property belongs to the `KinesisConsumerProperties`. This is really not a whole binder option.
Every binding may have its own checkpoint mode and, therefore, `checkPointInterval`.

Please, consider to refactor the code respectively.